docs/content/users/quickstart.md

```diff
@@ -458,21 +458,16 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
 === "OpenMage/Magento 1"
 
-    1. Download OpenMage from [release page](https://github.com/OpenMage/magento-lts/releases).
-    2. Make a directory for it, for example `mkdir ~/workspace/OpenMage` and change to the new directory `cd ~/workspace/OpenMage`.
-    3. Run [`ddev config`](../users/usage/commands.md#config) and accept the defaults.
-    4. Install sample data. (See below.)
-    5. Run [`ddev start`](../users/usage/commands.md#start).
-    6. Follow the URL to the base site.
-
-    You may want the [Magento 1 Sample Data](https://github.com/Vinai/compressed-magento-sample-data) for experimentation:
-
-    * Download Magento [1.9.2.4 Sample Data](https://github.com/Vinai/compressed-magento-sample-data/raw/master/compressed-magento-sample-data-1.9.2.4.tgz).
-    * Extract the download:
-        `tar -zxf ~/Downloads/compressed-magento-sample-data-1.9.2.4.tgz --strip-components=1`
-    * Import the example database `magento_sample_data_for_1.9.2.4.sql` with `ddev import-db --file=magento_sample_data_for_1.9.2.4.sql` to database **before** running OpenMage install.
-
-    OpenMage is a huge codebase, and we recommend [using Mutagen for performance](install/performance.md#mutagen) on macOS and traditional Windows.
+    ```bash
+    mkdir my-magento1-site && cd my-magento1-site
+    tag=$(curl -L "https://api.github.com/repos/OpenMage/magento-lts/releases/latest" | docker run -i --rm ddev/ddev-utilities jq -r .tag_name) && curl -L "https://github.com/OpenMage/magento-lts/releases/download/$tag/openmage-$tag.zip" -o openmage.zip
+    unzip ./openmage.zip && rm -f openmage.zip
+    ddev config --web-environment-add=MAGE_IS_DEVELOPER_MODE=1
+    ddev start
+    # Install sample data
+    ddev openmage-install
```

Comment:
I noticed that this command has some automation flags and we can execute it like this:
```
ddev openmage-install -q
```
But in this case, people won't know what the username/password is, so I decided to leave the interaction here.